api/inventorypb/agents.proto

```diff
@@ -712,6 +712,11 @@ message AddMongoDBExporterRequest {
   string authentication_database = 15;
   // Custom password for exporter endpoint /metrics.
   string agent_password = 16;
+  // List of colletions to get stats from. Can use *
+  repeated string stats_collections = 17;
+  // Collections limit. Only get Databases and collection stats if the total number of collections in the server
+  // is less than this value. 0: no limit
+  int32 collections_limit = 18;
```

Comment:
Also, it can be `uint32`.

Reply:
@BupycHuk what do you think?

Reply:
@percona-csalguero I remember that we had discussion about using -1 for no limit, right? 0 can be used for default limit like we do for MySQL